scapy/contrib/pnio_rpc.uts

```diff
@@ -5,6 +5,10 @@
 from scapy.contrib.dce_rpc import *
 from scapy.contrib.pnio_rpc import *
 
+= Check that we have UUIDs
+
+for v in RPC_INTERFACE_UUID.values():
```

Comment:
```suggestion
for v in itervalues(RPC_INTERFACE_UUID):
```
(You need to import `itervalues` from `scapy.modules.six`.

Reply:
Added.